### PR TITLE
HM 53 CICD stuff - Ready to merge!!

### PR DESCRIPTION
**Pull request by aschnapp**

### CODE REVIEW

**Comment by aschnapp** [comment]
Dont merge yet.

**Review by vperuzzi — APPROVED** [review]

File: server/README.md [hunk 1]
```diff
@@ -1,19 +1,8 @@
...
+sh run.sh --dev
 
 ## How to Run Tests
-```sh
-$ python -m pytest
-```
+sh run.sh --test
+
+## How to run lint
+sh run.sh --lint
```
Comment (aneuhold) [inline server/README.md:8]: It might be nice to have these in code blocks like they were before.

**Review by aneuhold — CHANGES_REQUESTED** [review]
Just 1 comment relating to the documentation.

**Comment by aneuhold** [comment]
Also, it might be good to know for the rest of the team how to run these tests and how to check where the builds are passing or failing. URLs and invites etc.

File: server/README.md (hunk 1 above)
Comment (aschnapp) [reply server/README.md:8]: Is reading run.sh not good enough? Why do we need a copy of the run.sh code in the readme? That also means if run.sh changes, we'll have to update the readme. It makes it not "dry"

**Comment by aschnapp** [comment]
> Also, it might be good to know for the rest of the team how to run these tests and how to check where the builds are passing or failing. URLs and invites etc.

It's in the details of "all checks passed, failed, whatever"

Screen Shot 2020-10-15 at 2 46 35 PM

File: server/README.md (hunk 1 above)
Comment (aneuhold) [reply server/README.md:8]: You already wrote the code. All that is needed is 3 back ticks. For example:

## How to run lint
```
sh run.sh --lint
```

That is all that is being said here.

Comment (aschnapp) [reply server/README.md:8]: Oh those three backticks. I thought you meant something else.

**Review by aneuhold — APPROVED** [review]
Looks good!

**Pull request merged**